src/hyperfine/benchmark.rs

```diff
@@ -244,23 +252,49 @@ pub fn run_benchmark(
     let (user_str, user_unit) = format_duration_unit(user_mean, None);
     let system_str = format_duration(system_mean, Some(user_unit));
 
+    let (mean_start, delta, mean_val, stddev_val, user_val, sys_val) = match options.output_style {
```

Comment:
The same here. I think we could just keep the old version and disable coloring globally.